content/doc/developer/security/index.adoc

```diff
@@ -96,6 +98,63 @@ To do this, write Jelly like this:
 NOTE: This is not to be confused with the +checkPermission+ invocation in your operation.
 Users can still hit the URL directly, so you still need to protect the operation itself, in addition to disabling the UI rendering
 
+=== Authentication methods
+
+In Jenkins the security engine that is used is Acegi Security (ancestor of Spring Security).
+Without any special plugins to manage authentication, an instance of Jenkins is packaged
+with the following authentication methods:
```

Comment:
"authentication method" may be a bit confusing, I would recommend "authentication ways" to avoid collisions with system terms (e.g. matrix auth).

Reply:
modified